New version: HomotopyOpt v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: HomotopyOpt
- Repository: https://github.com/matthiashimmelmann/HomotopyOpt.jl
- Created by: @matthiashimmelmann
- Version: v0.2.0
- Commit: bf8ec6443148e3b11244c33b7b1b15a1ecff8fa9
- Reviewed by: @matthiashimmelmann
- Reference: https://github.com/matthiashimmelmann/HomotopyOpt.jl/commit/bf8ec6443148e3b11244c33b7b1b15a1ecff8fa9#commitcomment-138464398
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd608f0b66b48fcaf6f30a2563715751025a67714e4e48ad33dad5f295814f1ba930f4475b2f6617e86007b4602c4fab6356bcfda0e40df9216b228cce252212e8a34414b2b342b7bb1447c6276af775be667b0b4e63694606c259f9da7e40e79fbfecd64fb0d15db6e4f2c09e583e3d4992e80a21efc28d17dda4416bc3dd06e82b0b960d6ecde81df041b5b36f4bd2586c14444af08fd0486b8f215a2e2d3373205b581e4a7a12afcd36549e4a4aa35aebcee947b5a3b6111a46c813b5c1606ca -->

matthiashimmelmann commented:
> [noblock]
> 
> Looks like there is a merge conflict.
> 
> @matthiashimmelmann Can you re-trigger Registrator? This will fix the merge conflict.

Sure, will do, thanks!

JuliaRegistrator committed:
New version: HomotopyOpt v0.2.0

UUID: 8c5e0f79-7402-43d4-9c5c-e1bde87b1895
Repo: https://github.com/matthiashimmelmann/HomotopyOpt.jl.git
Tree: 4e4cece45142daa92d0be1483c3a6a5728371296

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
